README.md

```diff
@@ -9,8 +9,6 @@
 [![GoDoc](https://godoc.org/github.com/ipfs/go-ipfs?status.svg)](https://godoc.org/github.com/ipfs/go-ipfs)
 [![Build Status](https://travis-ci.org/ipfs/go-ipfs.svg?branch=master)](https://travis-ci.org/ipfs/go-ipfs)
 
-> IPFS implementation in Go
```

Comment:
I'd leave this (unless there's a good reason to remove it).

Reply:
Isn't it obvious?

Reply:
I figure it can't hurt but you're right, it doesn't *really* have to be there.